### PR TITLE
Update point-in-time-recovery.md

### PR DESCRIPTION
**Pull request by lingamsandeep**
Update guidance on how to use PITR with xCluster.

### CODE REVIEW

File: docs/content/preview/manage/backup-restore/point-in-time-recovery.md [hunk 1]
```diff
@@ -215,17 +215,15 @@ Tracking issue: [12773](https://github.com/yugabyte/yugabyte-db/issues/12773)
 
 ### xCluster replication
 
-The combination of PITR and [xCluster replication](../../../explore/multi-region-deployments/asynchronous-replication-ysql/) is not fully tested, and is considered beta.
-
-xCluster does not replicate any commands related to PITR. If you have two clusters with replication between them, enable PITR on both ends separately. To restore, the following is the recommended procedure:
+xCluster does not replicate any commands related to PITR. If you have two clusters with replication between them, enable PITR on both ends independently. To perform a restore, the following is the recommended procedure:
```
Comment (aishwarya24) [inline docs/content/preview/manage/backup-restore/point-in-time-recovery.md:218]:
```suggestion
xCluster does not replicate any commands related to PITR. If you have two clusters with replication between them, enable PITR on both ends independently. You can perform a restore using the following recommended procedure:
```

File: docs/content/preview/manage/backup-restore/point-in-time-recovery.md [hunk 2]
```diff
@@ -215,17 +215,15 @@ Tracking issue: [12773](https://github.com/yugabyte/yugabyte-db/issues/12773)
...
 
 1. Stop application workloads and make sure there are no active transactions.
 1. Wait for replication to complete.
-1. Restore to the same time on both clusters.
+1. Delete xCluster replication from both sides.
```
Comment (aishwarya24) [inline docs/content/preview/manage/backup-restore/point-in-time-recovery.md:222]:
```suggestion
1. Delete xCluster replication from both clusters.
```

**Review by aishwarya24 — APPROVED** [review]
LGTM . Not sure if this is applicable for stable (2.18) version too. If so , it needs to be back-ported https://deploy-preview-19326--infallible-bardeen-164bc9.netlify.app/stable/manage/backup-restore/point-in-time-recovery/#xcluster-replication

**Review by ddhodge — APPROVED** [review]
LGTM. What versions does this apply to? Stable? Any others?

**Pull request merged**